### ISSUE TITLE
Leaving the help menu is quite inconsistent

### ISSUE THREAD

**Issue by phet-steele** [phetsims/capacitor-lab-basics#45]
@jessegreenberg you know how to decipher these hieroglyphics:
![20160407_090700](https://cloud.githubusercontent.com/assets/5863899/14356072/981b8a06-fca0-11e5-9bb3-f1aef5e5dcfc.jpg)

For everyone else, the gist of it is that the varying ways of leaving the help menu are drastically inconsistent. It would be nice @jessegreenberg if all 18 of these cases I've outlined behaved as the first one does. Every time you leave the help menu you should be brought back to the element that was selected before pressing "H". Also, maybe "H" could close the help menu, too? I've tried this in Win 10 Chrome/Edge. For phetsims/tasks#566

**Comment by jessegreenberg** [comment]
ff315af2b42049deeb6bceb6a48b2456330b3907 changes the implementation of the keyboard help panel so that it behaves like an accessible dialog.  Navigation is restricted to the panel unless the user presses  'escape' or closes the dialog with the 'Close' button.  Regardless of the method of closing, navigation returns to the previous active element.  Closing.
